tests/unittests/test_ds_identify.py

```diff
@@ -460,6 +319,37 @@ def test_wb_print_variables(self):
         for var in expected_vars:
             self.assertIn("{0}=".format(var), err)
 
+    @pytest.mark.xfail(reason="GH-4796")
+    def test_maas_not_detected_1(self):
+        """Don't incorrectly identify maas"""
+        config = "LXD-kvm-not-MAAS-1"
+        self._test_ds_found(config)
+
+    def test_maas_not_detected_2(self):
+        """Don't incorrectly identify maas
```

Comment:
Think it's worth putting an issue/commit reference in all of these tests (not just the xfail)? They just feel a bit random without the underlying context and if approaching otherwise I would think "why are we only testing maas here and not every other single datasource?"

Reply:
> Think it's worth putting an issue/commit reference in all of these tests

I think you missed the rest of the docstring?

>         The bug reported in 4794 combined with the previously existing bug
>         reported in 4796 made for very loose MAAS false-positives.

Or maybe I misunderstand what you are asking for.

Reply:
I've added more context to the tests.